app/controllers/kondos_controller.rb

```diff
@@ -3,11 +3,23 @@ class KondosController < ApplicationController
   skip_before_action :authenticate_user!, only: [:index, :show]
 
   def index
-    unless params[:search_kondos] == ""
-      @kondos = @kondos.where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")
-    else
-      @kondos = Kondo.all
+
+    #Renter's Page
+    if current_user.role == "renter"
```

Comment:
Similar to show controller.

Nothing wrong with this too but we can now be more fancy and take advantage of enum generated methods for our user roles like so:
```ruby
if current_user.renter?
```